# Error window size for scan_match_quality node
window_size: 50
